lib/mongo/auth.rb

```diff
@@ -70,6 +70,26 @@ def get(user)
       SOURCES[mechanism].new(user)
     end
 
+    # Raised when trying to authorize with an invalid configuration
+    #
+    # @since 2.11.0
```

Comment:
I'm assuming this is going to be released with 2.11.0, but let me know if that's wrong.